src/uk/gov/hmcts/contino/AppPipelineDsl.groovy

```diff
@@ -43,9 +43,16 @@ class AppPipelineDsl extends CommonPipelineDsl implements Serializable {
     config.parallelCrossBrowsers = browsers
   }
 
-  void enableSecurityScan(String urlExclusions = "", int timeout = 120) {
-    config.urlExclusions = urlExclusions
-    config.securityScanTimeout = timeout
+  void enableSecurityScan(Map<String, Object> params = [:]) {
+    def configuration = [
+        urlExclusions: "",
+        timeout: 120,
+        scanType: params.scanType ? params.scanType : ""
+    ] << params
+
+    config.securityScanUrlExclusions = configuration.urlExclusions
+    config.useFrontendSecurityScan = configuration.scanType
```

Comment:
you will also need to change a few other places
```suggestion
    config.securityScanType = configuration.scanType
```

Reply:
Updated